node/src/sub_lib/accountant.rs

```diff
@@ -94,8 +95,7 @@ pub struct AccountantSubs {
     pub report_routing_service_provided: Recipient<ReportRoutingServiceProvidedMessage>,
     pub report_exit_service_provided: Recipient<ReportExitServiceProvidedMessage>,
     pub report_services_consumed: Recipient<ReportServicesConsumedMessage>,
-    pub report_consuming_wallet_balances_and_qualified_payables:
-        Recipient<ConsumingWalletBalancesAndQualifiedPayables>,
+    pub report_consuming_wallet_balances_and_qualified_payables: Recipient<PayablePaymentSetup>,
```

Comment:
Field should be renamed. Also, you thought `PayablePaymentSetup` should be `PayablePaymentsSetup`.